src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -648,7 +478,6 @@ private ContextMenu createTabContextMenuFor(LibraryTab tab, KeyBindingRepository
     }
 
     public void addTab(LibraryTab libraryTab, boolean raisePanel) {
-        assert libraryTab != null;
```

Comment:
Either add JSpecify annotation or leave as is...  (IMHO). Chris might like `assert`, because it raises runtime exceptions during run.

Reply:
fixed, annotations added.